app.py

```diff
@@ -6,11 +6,13 @@
 import os
 import signal
 import sys
+import ssl
 
 from aiohttp import errors, web
 from jsonmodels import models, fields
 from jsonmodels.errors import ValidationError
 
+port=5000
```

Comment:
please don't do this.  Either make it properly configurable or leave it as it was.